docs/concepts/authentication.md

```diff
@@ -115,7 +115,7 @@ The behavior can be configured primarily by adding `activeOnPath` property for t
 
 This configuration would activate the specific identity provider, when the PWA initially routes to a specific url.
 
-An another possibility is to use the `OVERRIDE_IDENTITY_PROVIDERS` environment variable within the nginx deployment.
+Another possibility or the identity provider activation could be the usage of the `OVERRIDE_IDENTITY_PROVIDERS` environment variable within the nginx deployment.
```

Comment:
do you mean: Another possibility of the identity provider ... ?

Reply:
yes ;)